Docs/user_guide/modules/index.md

```diff
@@ -23,6 +23,11 @@ viewcontrollers.md
 volumerendering.md
 volumes.md
 ```
+## Wizards
```

Comment:
Since the category is consistent with the one set the module, introducing the category in the documentation makes sense.